src/less/components/links.less

```diff
@@ -84,9 +84,8 @@ a {
             }
         }
 
-        &:focus {
-            box-shadow: 0 0 0 1px @brand-secondary;
-            outline: none;
+        &:focus-visible {
```

Comment:
On line 27, change outline-color to be white or primary (links with dark background)